proxy/coco/src/state.rs

```diff
@@ -90,23 +90,21 @@ where
 ///   * Fails to verify `User`.
 ///   * Fails to set the default `rad/self` for this `PeerApi`.
 #[allow(clippy::single_match_else)]
-pub async fn init_owner(peer: &Peer<BoxedSigner>, name: String) -> Result<LocalIdentity, Error> {
+pub async fn init_owner(
+    peer: &Peer<BoxedSigner>,
+    payload: impl Into<payload::PersonPayload> + Send,
+) -> Result<LocalIdentity, Error> {
     match peer
         .using_storage(move |store| local::default(store))
         .await??
     {
         Some(owner) => Ok(owner),
         None => {
             let pk = keys::PublicKey::from(peer.signer().public_key());
+            let payload = payload.into();
```

Comment:
That was my initial solution. Unfortunately that would require an unconverted `payload` to be passed across a thread boundary, so it can be executed in a task. This is possible, but requires an unnecessary additional `Send` type constraint on the generic `payload` parameter, which is much noisier than this awkward conversion.